<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск файлов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Искать файлы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileExtensionLineEdit">
        <property name="text">
         <string>*.*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startSearchButton">
        <property name="text">
         <string>Начать поиск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Место поиска</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>C:/Qt/Projects/build-SearchProject-Desktop_Qt_5_13_2_MinGW_32_bit-Debug</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Обзор</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="notOlderCheckBox">
        <property name="text">
         <string>Не старше</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timeSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="timeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>дня(дней)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="fileSizeCheckBox">
        <property name="text">
         <string>Размер файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fileSizeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>&lt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>=</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fileSizeSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fileSizeType">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>КБ</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="resultsWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
